public/src/reimbursement-policy.md

```diff
@@ -2,107 +2,121 @@
+- Speedcubing Canada **must** only reimburse expenses that are reasonable and necessary for running Speedcubing Canada Supported Events as defined by the [Speedcubing Canada Supported Events Policy](https://www.speedcubingcanada.org/documents/supported-events-policy.pdf).
+  - Speedcubing Canada reserves the right to provide partial reimbursement of an allowed expense if deemed appropriate.
+  - Speedcubing Canada reserves the right to deny unnecessary or unreasonable reimbursement requests.
+- Speedcubing Canada **must** only reimburse expenses as is allowed for Canadian not-for-profit corporations.
+- Speedcubing Canada **must** only reimburse Director(s), Officer(s), Member(s) or other individual(s) and/or group(s) who are operating as unpaid volunteers to run Speedcubing Canada Supported Events.
 
 ### Venues and associated costs
 
-- Reimbursement requests can be made for venues used for a Speedcubing Canada event.
+- Reimbursement requests are allowed to be made for venues used for Speedcubing Canada Supported Events.
 - Expenses covered by this policy include:
   - Venue rental fees
   - Non-refundable damage deposits
   - Special event liability insurance
   - Wired or wireless internet access
-  - Audio-visual
+  - Audio-visual services or equipment rental
   - Risers
   - Tables
   - Chairs
   - Belt barriers or rope and stanchions
 
 ### Operating expenses
 
-- Reimbursement requests can be made for operating expenses of a Speedcubing Canada event.
+- Reimbursement requests are allowed to be made for operating expenses of Speedcubing Canada Supported Events.
 - Expenses covered by this policy include:
   - World Cube Association Competition Dues
-  - Stripe Fees
 
 ### Equipment
 
-- Reimbursement requests can be made for equipment used for a Speedcubing Canada event.
+- Reimbursement requests are allowed to be made for equipment used for Speedcubing Canada Supported Events.
 
-#### Type 1
+#### Type 1: timing equipment
 
-- Equipment used during scrambling and solving in an official World Cube Association competition.
 - Expenses covered by this policy include:
   - StackMat timers
   - StackMat mats
   - Tournament displays
   - Tournament display data cords
+  - Tournament display power adaptors
   - Tournament display stands
   - Stopwatches
-  - Batteries
-  - O-rings
+  - O-rings and other button protectors
+
+#### Type 2: non-timing equipment
+
+- Expenses covered by this policy include:
   - Cube covers
-  - Stationary (e.g. pens, markers)
-  - Tri-folds
-  - Laptops and tablets (only if necessary for scrambling stations)
-  - Harmonica holders (only if necessary for 4x4x4 Blindfolded, 5x5x5 Blindfolded or 3x3x3 Multi-Blind)
-  - Cardstock and paper (e.g. for scorecards, FMC, 3x3x3 Blindfolded, 4x4x4 Blindfolded, 5x5x5 Blindfolded or 3x3x3 Multi-Blind)
+  - Scrambling station blockers (e.g. tri-folds, pipe and drape)
+  - Blind blockers and harmonica holders
```

Comment:
```suggestion
  - Sight blockers and harmonica holders
```
Term used in the [regs](https://www.worldcubeassociation.org/regulations/#B4c).

Reply:
Fixed